README.aspnetcore.md

```diff
@@ -15,10 +15,14 @@ These images contain the runtime only. Use [`microsoft/aspnetcore-build-nightly`
 - `1.1.2`, `1.1`, `1`
     - [`1.1.2-jessie` (*Dockerfile*)](https://github.com/aspnet/aspnet-docker/blob/dev/1.1/jessie/runtime/Dockerfile)
     - [`1.1.2-nanoserver` (*Dockerfile*)](https://github.com/aspnet/aspnet-docker/blob/dev/1.1/nanoserver/runtime/Dockerfile)
-- `2.0.0`, `2.0`, `2`, `latest`
+- `2.0.0`, `2.0`
```

Comment:
It doesn't update the actual readme file - it just writes to the console.  Could be made better but this was what I considered the min bar 😄.  Feel free to make a contribution to the tool if you have ideas.